src/psyclone/psyir/symbols/symbol.py

```diff
@@ -248,10 +248,18 @@ def resolve_type(self):
                   examining the Container from which it is imported.
         :rtype: subclass of :py:class:`psyclone.psyir.symbols.Symbol`
 
+        :raises SymbolError: if the type could not be resolved.
         '''
         if self.is_import:
             extern_symbol = self.get_external_symbol()
+            # pylint: disable=import-outside-toplevel
```

Comment:
Just to be precise, could you do `disable-next=...` here and below.

Reply:
but I removed one of them as I don't need the type equality check anymore